### PR TITLE
Generated model tests and add validations

### PR DESCRIPTION
**Pull request by dlanoronald123**
Generated model tests on transactions, brokers_stocks, buyer_stocks,  and stocks.
Add validations on models for the tests to pass.

### CODE REVIEW

**Review by villanuevajamesfvillanueva — APPROVED** [review]

File: app/models/broker_stock.rb [hunk 1]
```diff
@@ -1,2 +1,7 @@
 class BrokerStock < ApplicationRecord
+  validates :user_id, presence: true
+  validates :stock_id, presence: true
```
Comment (analyn-cajocson) [inline app/models/broker_stock.rb]: No need to validates foreign_key, just define the database association of `BrokerStock` with `User` and `Stock` model

File: app/models/buyer_stock.rb [hunk 2]
```diff
@@ -1,2 +1,7 @@
 class BuyerStock < ApplicationRecord
+  validates :user_id, presence: true
+  validates :stock_id, presence: true
```
Comment (analyn-cajocson) [inline app/models/buyer_stock.rb]: Same comment as above

File: app/models/transact.rb [hunk 3]
```diff
@@ -1,2 +1,7 @@
 class Transact < ApplicationRecord
+  validates :broker_id, presence: true
+  validates :buyer_id, presence: true
```
Comment (analyn-cajocson) [inline app/models/transact.rb]: Same comment as above

File: spec/models/stock_spec.rb [hunk 4]
```diff
@@ -1,5 +1,23 @@
...
+    stocks.companyname = nil
+    expect(stocks).not_to be_valid
+  end
 end
+
+# describe 'list' do
+#   it 'returns active stocks from IEX Ruby Client' do
+#     expect(described_class.list).to be_an_instance_of(Array)
+#   end
+# end
```
Comment (analyn-cajocson) [inline spec/models/stock_spec.rb]: remove unnecessary comments

File: spec/models/buyer_stock_spec.rb [hunk 5]
```diff
@@ -1,5 +1,42 @@
...
+#   it 'is not valid without a price' do
+#     buyers_stocks.price = nil
+#     expect(buyers_stocks).not_to be_valid
+#   end
+
+#   it 'is not valid without a broker_id' do
+#     buyers_stocks.broker_id = nil
+#     expect(buyers_stocks).not_to be_valid
+#   end
+# end
```
Comment (analyn-cajocson) [inline spec/models/buyer_stock_spec.rb]: remove unnecessary comments

**Review by analyn-cajocson — CHANGES_REQUESTED** [review]

File: app/models/broker_stock.rb (hunk 1 above)
Comment (dlanoronald123) [reply app/models/broker_stock.rb]: Noted on this Ma'am. Will update the model po. Thank you.

**Review by analyn-cajocson — APPROVED** [review]

**Pull request merged**